asyncio_run_in_process/_utils.py

```diff
@@ -51,3 +56,30 @@ def receive_pickled_value(stream: BinaryIO) -> Any:
     serialized_len = int.from_bytes(len_bytes, "big")
     serialized_result = read_exactly(stream, serialized_len)
     return cloudpickle.loads(serialized_result)
+
+
+class RemoteTraceback(Exception):
+
+    def __init__(self, tb: str) -> None:
+        self.tb = tb
+
+    def __str__(self) -> str:
+        return self.tb
+
+
+class RemoteException(Exception):
+    def __init__(self, exc: BaseException, tb: TracebackType) -> None:
+        self.tb = (
+            f'\n""" (exception from process: {os.getpid()})\n'
+            f"{''.join(traceback.format_exception(type(exc), exc, tb))}"
+            '"""'
+        )
+        self.exc = exc
+
+    def __reduce__(self) -> Any:
```

Comment:
A docstring could be helpful here, I didn't know what this was doing until I looked at the test.